src/identity/manager.rs

```diff
@@ -414,6 +415,19 @@ impl SecretManager<CaClient> {
     }
 }
 
+impl SecretManager<CustomSigner> {
+    pub async fn origin(cfg: crate::config::Config) -> Result<Self, anyhow::Error> {
```

Comment:
what does "origin" mean?

Reply:
Associated function - https://doc.rust-lang.org/rust-by-example/fn/methods.html.

Reply:
I know it is a function, I just do not get the name?

Reply:
oh I saw the same name at many places in rust example codes as a replacement of `new`. Do you think I should update it?